stylesheets/components/cards.css

```diff
@@ -1,204 +1,151 @@
-  border-radius: var(--border-rd-6);
+.card {
+  border: 1.35px solid var(--dark-pastel-grey);
   border-radius: 8px;
-  /* overflow: */
-  /* box-shadow: 0px 0px 15px hsl(0, 0%, 80%); */
-
-width:18.5rem;
-}
-.card{
- 
-    /* width:0rem; */
-
-}
-.card-container:hover{
-    box-shadow: 4px 4px 15px var(--charcoal-grey);  
-     transition: 0.3s ease-in;
-}
-.card{
-    gap:var(--spacing-xs);
-    position: relative;
-}
-.card-img-container{
-    height:17rem;
-    /* height: fit-content; */
-    /* overflow: hidden; */
-    /* border-radius:16px; */
-}
-.card-img{
-    width: 100%; 
-    /* height:100%; */
-    /* max-height: 100%;  */
-    /* margin: 0 auto; */
-}
-.icon-top-right{
-    color:hsl(0, 0%, 60%);
-    position: absolute;
-    top:10px;
-    right:var(--spacing-xs);
-}
-.badge{
-    background-color: #075729;
-  /* background-color:var(--grape-fruit); */
-  color:white;
-  padding:0.3rem 0.65rem;
-  /* border-radius: var(--border-rd-6); */
-  border-top-right-radius: 2px;
-  border-bottom-right-radius: 2px;
+  background-color: var(--white-color);
+  gap: 1rem;
+  max-width: 18.5rem;
+  position: relative;
+}
+.card:hover {
+  box-shadow: 4px 4px 15px var(--charcoal-grey);
+  transition: 0.4s ease-in;
+}
+.card-header {
+  max-height: 17rem;
+}
+.card-img {
+  border-top-left-radius: 8px;
+  border-top-right-radius: 8px;
+}
+.icon-top-right {
+  color: var(--dark-pastel-grey);
   position: absolute;
-  top:10px;
-  left:-3px;
-  font-size:0.8rem;
-  letter-spacing: 0.3px;
+  top: 10px;
+  right: var(--spacing-xs);
+}
+.icon-dismiss {
+  color: var(--charcoal-grey);
+}
+.card-body {
+  margin: 0 1rem 1rem 1rem;
+  gap: 0.25rem;
+}
+.card-title {
+  font-size: 1.4rem;
 }
-.rating-icon-wrapper{
-font-size: 2px;
+.card-text {
+  color: var(--text-color);
+  font-size: 0.9rem;
+  font-weight: 300;
 }
-.card-body{
-    margin-top:0.3rem;
-  margin-left:1rem;
-  margin-right:1rem;
-  margin-bottom: 1rem;
-  gap:0.2rem;
-
+.card-section {
+  justify-content: space-between;
+  flex-wrap: wrap;
 }
-.card-title{
-    font-size:1.4rem;
+.card-price {
+  gap: var(--spacing-xs);
 }
-.card-text{
-    color:var(--text-color);
-    font-size: 0.9rem;
-    font-weight: 300;
+.card-current-price {
+  font-size: 1.2rem;
+  font-weight: 600;
 }
-.card-section{
-    justify-content: space-between;
+.card-price-striked {
+  text-decoration: line-through var(--gravel-color);
 }
-.card-price{
-    gap:var(--spacing-xs);
+.card-rating {
+  gap: 0.3rem;
 }
-.card-current-price{
-    font-size:1.2rem;
-    font-weight: 600;
+.rating-icon-wrapper {
+  gap: 0.1rem;
 }
-.card-price-striked{
-    text-decoration: line-through;
+.rating-icon {
+  color: var(--yellow-dark);
+  font-size: 0.7rem;
 }
-.card-rating{
-    gap:0.3rem;
+.rating-count {
+  font-size: 0.7rem;
 }
-.rating-icon-wrapper{
-    gap:0.1rem;
+.card-actions {
+  margin-top: 1.5rem;
+  gap: 1rem;
 }
-.rating-icon{
-    color:var(--yellow-dark);
-    font-size:12px;
```

Comment:
use absolute value for more responsive

Reply:
This was done intentionally, as I wanted the review icon to be of a fixed size